config/samples/metallb.io_v1alpha1_addresspool.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: metallb.io/v1alpha1
+kind: AddressPool
+metadata:
+  name: addresspool-sample1
+  namespace: metallb-system
+spec:
+  address-pool:
+    name: default
+    protocol: layer2
+    addresses:
+    - 172.18.0.100-172.18.0.200
+
+---
+
+apiVersion: metallb.io/v1alpha1
+kind: AddressPool
+metadata:
+  name: addresspool-sample2
+  namespace: metallb-system
+spec:
+  address-pool:
+    name: gold
+    protocol: layer2
+    addresses:
+    - 172.20.0.100-172.20.0.200
+    auto-assign: false
+
+---
+
+apiVersion: metallb.io/v1alpha1
+kind: AddressPool
+metadata:
+  name: addresspool-sample3
+  namespace: metallb-system
+spec:
+  address-pool:
+    name: silver
+    protocol: layer2
+    addresses:
+    - 172.21.0.100-172.21.0.200
```

Comment:
CIDR notation is supported too, right?  Can you add an example like that?  `10.10.10.0/24` or something

and how about an IPv6 example, too?

Reply:
yeah I add one for CIDR but I haven't seen any configs for IPv6 on metallb so not sure if that is fully supported today ?

Reply:
yeah, I can't find any examples in the metallb repo, but I can see in the code that IPv6 is supported.  Don't worry about that part for now.

Reply:
i added v6 sample